crates/sncast/tests/e2e/account/delete.rs

```diff
@@ -253,6 +253,44 @@ pub async fn test_happy_case_without_network_args() {
     let _ = tokio::fs::remove_file("temp_scarb6.toml").await;
 }
 
+#[tokio::test]
+pub async fn test_happy_case_with_yes_flag() {
+    // Creating dummy accounts test file
+    create_dummy_accounts_file("temp_accounts7.json").await;
+    create_dummy_scarb_file("temp_scarb7.toml").await;
```

Comment:
Not in the scope of this PR, but what do you think about changing these functions to use `TempDir` and to return paths to created files? Then we wouldn't need to remove files at the end.

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/1247